packages/manager/modules/exchange/src/translations/Messages_fr_FR.json

```
@@ -1565,5 +1578,20 @@
   "exchange_wizardHostedCreation_configureDNSZone_manual_dialog_warning": "Attention",
   "exchange_wizardHostedCreation_lastDialog_closeButton": "Finaliser",
   "exchange_table_manage_entry": ": gérer",
-  "sharepoint_activate_which_service_label": "Serveur Exchange à associer"
+  "sharepoint_activate_which_service_label": "Serveur Exchange à associer",
+  "exchange_mfa_description_ENABLE": "Êtes-vous sûr de vouloir activer la double authentification pour le compte <strong>{{account}}</strong>?",
```

I suggest this change for the next four occurrences:

```suggestion
  "exchange_mfa_description_ENABLE": "Êtes-vous sûr(e) de vouloir activer la double authentification pour le compte <strong>{{account}}</strong>?",
```